webapp_v2/src/components/singlestudy/NavHeader.tsx

```diff
@@ -264,6 +267,17 @@ function NavHeader(props: PropTypes) {
               <HistoryOutlinedIcon />
             </Button>
           )}
+          {!isExplorer && (
```

Comment:
use the HistoryOutlinedIcon (previous button) 
and check if it's a variant (study.type == "variantstudy")

this button should appear in explore mode as well as dashboard mode